methods.py

```diff
@@ -259,14 +259,11 @@ def generate_version_header(module_version_string=""):
 
 
 def parse_cg_file(fname, uniforms, sizes, conditionals):
-
```

Comment:
I wonder why `black` didn't fix those until I actually modified the file locally and had it touched by my pre-commit hook.